uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/manager-leave/components/manager-leave-requests_test.js

```diff
@@ -892,6 +892,17 @@
         });
       });
 
+      describe('when new leave request is updated', function() {
+        beforeEach(function() {
+          spyOn(controller,'refresh').and.callThrough();
```

Comment:
In this case `.and.callThrough()` is not required as you are not testing anything which is happening inside refresh.